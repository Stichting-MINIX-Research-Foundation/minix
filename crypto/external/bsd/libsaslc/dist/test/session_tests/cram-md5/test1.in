CRAM-MD5
2
AUTHID joe
PASSWD tanstaaftanstaaf
<1896.697170952@postoffice.example.net>


