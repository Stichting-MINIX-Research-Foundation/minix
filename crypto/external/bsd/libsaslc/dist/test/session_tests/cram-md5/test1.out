mechanism: session:
joe 3dbc88f0624776a737b39093f6eb6427
